### PR TITLE
[bext-ut] Update to 1.1.9

### PR DESCRIPTION
**Pull request by chausner**
**Describe the pull request**

- #### What does your PR fix?  
  Updates bext-ut to 1.1.9

- #### Which triplets are supported/not supported? Have you updated the [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt)?  
  As before, Yes

- #### Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?  
  Yes

- #### If you have added/updated a port: Have you run ./vcpkg x-add-version --all and committed the result?  
  Yes

### CODE REVIEW

File: ports/bext-ut/portfile.cmake [hunk 1]
```diff
@@ -1,31 +1,24 @@
...
     OPTIONS
         -DBOOST_UT_BUILD_BENCHMARKS=OFF
         -DBOOST_UT_BUILD_EXAMPLES=OFF
         -DBOOST_UT_BUILD_TESTS=OFF
 )
-vcpkg_install_cmake()
+vcpkg_cmake_install()
 vcpkg_copy_pdbs()
-vcpkg_fixup_cmake_targets(CONFIG_PATH lib/cmake/ut TARGET_PATH share/ut)
+vcpkg_cmake_config_fixup(PACKAGE_NAME ut CONFIG_PATH lib/cmake/ut-1.1.8)
```
Comment (chausner) [inline ports/bext-ut/portfile.cmake:18]: Version 1.1.8 is intentionally used here, they forgot to bump the version.

File: ports/bext-ut/portfile.cmake [hunk 2]
```diff
@@ -1,31 +1,24 @@
...
 
-configure_file("${LICENSE_FILE}" "${CURRENT_PACKAGES_DIR}/share/${PORT}/copyright" COPYONLY)
+file(INSTALL "${SOURCE_PATH}/LICENSE.md" DESTINATION "${CURRENT_PACKAGES_DIR}/share/{PORT}" RENAME copyright)
```
Comment (Thomas1664) [inline ports/bext-ut/portfile.cmake:20]: Please put this at the end of the file

Comment (chausner) [reply ports/bext-ut/portfile.cmake]: Done: a8942d227fd4f58acba55803f641d672b4738f34

**Comment by chausner** [comment]
CI fails on all platforms with
```
-- The CXX compiler identification is MSVC 19.31.31106.2
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: C:/Program Files/Microsoft Visual Studio/2022/Enterprise/VC/Tools/MSVC/14.31.31103/bin/Hostx64/x64/cl.exe - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Downloading CPM.cmake to D:/buildtrees/bext-ut/x64-windows-rel/cmake/CPM_0.31.1.cmake
-- CPM: adding package PackageProject.cmake@1.6.0 (v1.6.0)
CMake Error at D:/downloads/tools/cmake-3.22.2-windows/cmake-3.22.2-windows-i386/share/cmake-3.22/Modules/ExternalProject.cmake:2666 (message):
  error: could not find git for clone of packageproject.cmake-populate
Call Stack (most recent call first):
  D:/downloads/tools/cmake-3.22.2-windows/cmake-3.22.2-windows-i386/share/cmake-3.22/Modules/ExternalProject.cmake:3716 (_ep_add_download_command)
  CMakeLists.txt:23 (ExternalProject_Add)
```

It appears that git is not installed system-wide on the CI runners and CMake cannot find it. I have added all platforms to `ci.baseline.txt`.

**Comment by Thomas1664** [comment]
> CI fails on all platforms with
> ```
> -- The CXX compiler identification is MSVC 19.31.31106.2
> -- Detecting CXX compiler ABI info
> -- Detecting CXX compiler ABI info - done
> -- Check for working CXX compiler: C:/Program Files/Microsoft Visual Studio/2022/Enterprise/VC/Tools/MSVC/14.31.31103/bin/Hostx64/x64/cl.exe - skipped
> -- Detecting CXX compile features
> -- Detecting CXX compile features - done
> -- Downloading CPM.cmake to D:/buildtrees/bext-ut/x64-windows-rel/cmake/CPM_0.31.1.cmake
> -- CPM: adding package PackageProject.cmake@1.6.0 (v1.6.0)
> CMake Error at D:/downloads/tools/cmake-3.22.2-windows/cmake-3.22.2-windows-i386/share/cmake-3.22/Modules/ExternalProject.cmake:2666 (message):
>   error: could not find git for clone of packageproject.cmake-populate
> Call Stack (most recent call first):
>   D:/downloads/tools/cmake-3.22.2-windows/cmake-3.22.2-windows-i386/share/cmake-3.22/Modules/ExternalProject.cmake:3716 (_ep_add_download_command)
>   CMakeLists.txt:23 (ExternalProject_Add)
> ```
> 
> It appears that git is not installed system-wide on the CI runners and CMake cannot find it. It builds successfully on my local machine. I have added all platforms to `ci.baseline.txt`.

We can try to patch out the use of git

**Comment by dg0yt** [comment]
> -- Downloading CPM.cmake to D:/buildtrees/bext-ut/x64-windows-rel/cmake/CPM_0.31.1.cmake
> -- CPM: adding package PackageProject.cmake@1.6.0 (v1.6.0)

It is not just git but an entire package manager. This must be avoided.

**Comment by chausner** [comment]
> It is not just git but an entire package manager. This must be avoided.

Thanks for the hint, it turned out to be easy to disable it.

**Review by Adela0814 — APPROVED** [review]

**Comment by strega-nil-ms** [comment]
Thanks!

**Pull request merged**